_docs-sources/guides/build-it-yourself/achieve-compliance/deployment-walkthrough/manual-steps.md

```diff
@@ -0,0 +1,137 @@
+---
+pagination_label: Deployment Walkthrough
+---
+
+# Manual steps
+
+Now that your infrastructure is up and running, there are some manual steps necessary before the resources meet the CIS AWS Benchmark recommendations.
+
+## 1. Revoke Gruntwork's access
+
+Now that your infrastructure is deployed, Gruntwork doesn't need access to it anymore. The access is given through an IAM role called `GruntworkAccessRole` in each of the accounts through the `AdministratorAccess` policy. Use the `gruntwork` CLI to delete the IAM role and revoke access in each account by [following these steps](https://github.com/gruntwork-io/gruntwork#revoking-access-to-aws):
+
+```bash
+gruntwork aws revoke \
+            --account "dev" \
+            --account "stage" \
+            --account "prod" \
+            --account "security" \
+            --account "shared" \
+            --account "logs"
+```
+
+To revoke Gruntwork's access from the "current" account—the one you are authenticated to in the terminal—use the name
```

Comment:
I still think we should offer folks an alternative to needing to use the `gruntwork` CLI just to revoke our account access, because it's annoying to have to go and install a new binary just to accomplish a one-off task.

If they understand that our access is only possible due to the presence of the `GruntworkAccountAccessRole` - then anyone who understands IAM roles will know they can also just delete that role.

Reply:
Good point. Do you think this is clear from the docs? It mentions what the CLI command does.

> Use the `gruntwork` CLI to delete the IAM role and revoke access in each account by [following these steps]